### PR TITLE
Performance audit

### PR DESCRIPTION
**Pull request by jaimevillegas**
# Performance improvement 
## SELECT COUNT(*) FROM visits where animal_id = 4;
**Before**

Captura de pantalla 2023-05-08 121725
**After**
Captura de pantalla 2023-05-08 121948

## SELECT * FROM visits where vet_id = 2;
**Before**

Captura de pantalla 2023-05-08 121744

**After**
Captura de pantalla 2023-05-08 142203

## SELECT * FROM owners where email = 'owner_18327@mail.com';
**Before**
Captura de pantalla 2023-05-08 121744
**After** 
Captura de pantalla 2023-05-08 122341

### CODE REVIEW

**Comment by mukhammadrizooff** [comment]
Hi @jaimevillegas, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, and your team member should participate in this project because this project is to complete a group!

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by mukhammadrizooff** [comment]
- [ ] [OPTIONAL OPINION] image

File: queries.sql [hunk 1]
```diff
@@ -50,29 +50,26 @@ SELECT animals.name, species.name, owners.full_name FROM animals JOIN owners ON
...
 /* How many visits were with a vet that did not specialize in the animal's species? */
 SELECT COUNT(visits.animal_id) FROM visits JOIN animals ON visits.animal_id = animals.id JOIN vets ON visits.vet_id = vets.id JOIN specializations ON vets.id = specializations.vet_id WHERE specializations.species_id != animals.species_id;
-
 /* What speciality should Maisy Smith consider gretting? Look for the species she gets the most */
-SELECT COUNT(species.name) as Count, species.name FROM visits JOIN vets ON visits.vet_id = vets.id JOIN animals ON visits.animal_id = animals.id JOIN species ON animals.species_id = species.id WHERE vets.name = 'Maisy Smith' GROUP BY species.name;
\ No newline at end of file
+SELECT COUNT(species.name) as Count, species.name FROM visits JOIN vets ON visits.vet_id = vets.id JOIN animals ON visits.animal_id = animals.id JOIN species ON animals.species_id = species.id WHERE vets.name = 'Maisy Smith' GROUP BY species.name;
+/* EXPLAIN ANALYZE */
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animal_id = 4;
+EXPLAIN ANALYZE SELECT * FROM visits where vet_id = 2;
```
Comment (mukhammadrizooff) [inline queries.sql:74]: - [x] [REQUIRED] Good job so far! The second `screenshot` should show how much time the query takes `AFTER` the improvements introduced by you - The `execution` time should be significantly `lower` than 1 sec (`1000 ms`). After the implementation, please `update` the screenshot with the `AFTER` result on your `Pull Request` summary 🙏🏼 

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @TEAM, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Correct GitFlow ✔️ 
- Good implementations ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (jaimevillegas) [reply queries.sql:74]: Hi, @mukhammadrizooff!

Thank you for your review! We have worked on resolving that, and now it's done!

**Review by iambenkis — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @iambenkis in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**